src/lib/log.js

```diff
@@ -120,11 +120,11 @@ const Log = class {
    * Sends a report to third party error logging service
    *
    * @param {Error} err - an error to capture
-   * @param {Object} data - any extra data to send along with error
+   * @param {Object} [data] - any extra data to send along with error
    * @returns {Void}
    */
-  report (err, data) {
-    if (this.reporter != null) {
+  report (err: Error, data: ?Object) {
```

Comment:
The question mark makes it optional. `any` allows any type, but it will error if nothing is given.